metrics/ci_integrations/docs/diagrams/ci_integrations_config_validator_destination_class_diagram.puml

```diff
@@ -1,54 +1,64 @@
 @startuml ci_integrations_config_validator_class_diagram
```

Comment:
```suggestion
@startuml config_validator_class_diagram
```
Does it make sense to rename this file since it is already located in the `ci_integrations` package and I believe we can make its name a bit shorter and more readable? Please, apply this suggestion to all diagram files.